tikv-client/src/main/java/com/pingcap/tikv/region/RegionStoreClient.java

```diff
@@ -105,8 +105,7 @@ public int getValue() {
   // lockResolverClient, after implements the
   // write implementation of tispark, we can change
   // it to private
-  @VisibleForTesting
-  public final LockResolverClient lockResolverClient;
+  @VisibleForTesting public final LockResolverClient lockResolverClient;
```

Comment:
please undo this change

Reply:
Why，it is formatted this way.

Reply:
When you add an annotation before a variable/constant, google format state that they should be in the same line.

Reply:
Interesting - -